types/rusha/rusha-tests.ts

```diff
@@ -0,0 +1,17 @@
+import rusha from 'rusha';
```

Comment:
```suggestion
import Rusha = require('rusha');
```
Notice `Rusha`, be consistent with documented conventions, this is a class-like object

Reply:
Done!